test/unittests/commands/xmlupload/stash/test_create_info_for_graph_from_deserialised_data.py

```diff
@@ -0,0 +1,172 @@
+import pytest
+
+from dsp_tools.commands.xmlupload.stash.create_info_for_graph_from_deserialised_data import _get_stand_off_links
+from dsp_tools.commands.xmlupload.stash.create_info_for_graph_from_deserialised_data import _process_link_value
+from dsp_tools.commands.xmlupload.stash.create_info_for_graph_from_deserialised_data import _process_one_resource
+from dsp_tools.commands.xmlupload.stash.create_info_for_graph_from_deserialised_data import _process_richtext_value
+from dsp_tools.commands.xmlupload.stash.create_info_for_graph_from_deserialised_data import (
+    create_info_for_graph_from_data,
+)
+from dsp_tools.commands.xmlupload.stash.graph_models import LinkValueLink
+from dsp_tools.commands.xmlupload.stash.graph_models import StandOffLink
+from dsp_tools.utils.xml_parsing.models.data_deserialised import DataDeserialised
+from dsp_tools.utils.xml_parsing.models.data_deserialised import KnoraValueType
+from dsp_tools.utils.xml_parsing.models.data_deserialised import MigrationMetadata
+from dsp_tools.utils.xml_parsing.models.data_deserialised import ResourceDeserialised
+from dsp_tools.utils.xml_parsing.models.data_deserialised import ValueInformation
+
+
+@pytest.fixture
+def link_value() -> ValueInformation:
+    return ValueInformation("prop", "res_id_target", KnoraValueType.LINK_VALUE, [])
+
+
+@pytest.fixture
+def text_value_with_link() -> ValueInformation:
+    return ValueInformation(
+        "prop",
+        'This text contains a link: <a class="salsah-link" href="IRI:res_id_target:IRI">res_id_target</a>',
```

Comment:
To make this more robust, I'd choose a different text for the displayed link text:
```suggestion
        'This text contains a link: <a class="salsah-link" href="IRI:res_id_target:IRI">target resource</a>',
```